torchmetrics/image/inception.py

```diff
@@ -0,0 +1,174 @@
+              an ``[N,d]`` matrix where ``N`` is the batch size and ``d`` is the feature size.
+
+        splits: integer determining how many splits the inception score calculation should be split among
+
+        compute_on_step:
+            Forward only calls ``update()`` and return ``None`` if this is set to ``False``.
+        dist_sync_on_step:
+            Synchronize metric state across processes at each ``forward()``
+            before returning the value at the step
+        process_group:
+            Specify the process group on which synchronization is called.
+            default: ``None`` (which selects the entire world)
+        dist_sync_fn:
+            Callback that performs the allgather operation on the metric state. When ``None``, DDP
+            will be used to perform the allgather
+
+    [1] Improved Techniques for Training GANs
+    Tim Salimans, Ian Goodfellow, Wojciech Zaremba, Vicki Cheung, Alec Radford, Xi Chen
+    https://arxiv.org/abs/1606.03498
+
+    [2] GANs Trained by a Two Time-Scale Update Rule Converge to a Local Nash Equilibrium,
+    Martin Heusel, Hubert Ramsauer, Thomas Unterthiner, Bernhard Nessler, Sepp Hochreiter
+    https://arxiv.org/abs/1706.08500
+
+    Raises:
+        ValueError:
+            If ``feature`` is set to an ``str`` or ``int`` and ``torch-fidelity`` is not installed
+        ValueError:
+            If ``feature`` is set to an ``str`` or ``int`` and not one of ['logits_unbiased', 64, 192, 768, 2048]
+
+    Example:
+        >>> import torch
+        >>> _ = torch.manual_seed(123)
+        >>> from torchmetrics import IS
+        >>> inception = IS()  # doctest: +SKIP
+        >>> # generate some images
+        >>> imgs = torch.randint(0, 255, (100, 3, 299, 299), dtype=torch.uint8)  # doctest: +SKIP
+        >>> inception.update(imgs)  # doctest: +SKIP
+        >>> inception.compute()  # doctest: +SKIP
+        (tensor(1.0569), tensor(0.0113))
+
+    """
+
+    def __init__(
+        self,
+        feature: Union[str, int, torch.nn.Module] = 'logits_unbiased',
+        splits: int = 10,
+        compute_on_step: bool = False,
+        dist_sync_on_step: bool = False,
+        process_group: Optional[Any] = None,
+        dist_sync_fn: Callable = None
+    ):
+        super().__init__(
+            compute_on_step=compute_on_step,
+            dist_sync_on_step=dist_sync_on_step,
+            process_group=process_group,
+            dist_sync_fn=dist_sync_fn,
+        )
+
+        rank_zero_warn(
+            'Metric `IS` will save all extracted features in buffer.'
+            ' For large datasets this may lead to large memory footprint.', UserWarning
+        )
+
+        if isinstance(feature, (str, int)):
+            if not _TORCH_FIDELITY_AVAILABLE:
+                raise ValueError(
+                    'IS metric requires that Torch-fidelity is installed.'
+                    'Either install as `pip install torchmetrics[image-quality]`'
```

Comment:
```suggestion
                    'Either install as `pip install torchmetrics[image]`'
```
see: https://github.com/PyTorchLightning/metrics/blob/0ad5db63e99dbda32b0e75f666ae77efc71ad1de/setup.py#L29